lib/Dialect/TTNN/Transforms/Passes.cpp

```diff
@@ -150,6 +163,7 @@ class ConvertTTIRToTTNN
              TTIRToTTNNOpRewriter<ttir::ReluOp, ReluOp>,
              TTIRToTTNNBinaryOpRewriter<ttir::MatmulOp, MatmulOp>,
              TTIRToTTNNReductionOpRewriter<ttir::SumOp, SumOp>,
+             TTIRToTTNNSoftmaxOpRewriter<ttir::SoftmaxOp, SoftmaxOp>,
```

Comment:
I'm not sure this one needs to be templated since it's already specialized for softmax.

Reply:
Yes, makes sense, as it won't be reused, I will specialize it only for softmax. Thanks, Nick, good catch!